Sample/SampleDelegate/SampleDelegate/AppDelegate.swift

```diff
@@ -100,62 +100,62 @@ extension AppDelegate: RemoteNotificationCapable {
 
 extension AppDelegate: UserNotificationCapable {
     func requestedUserNotificationSettings() -> UIUserNotificationSettings {
-        return UIUserNotificationSettings(forTypes: [.Badge, .Sound, .Alert], categories: nil)
+        return UIUserNotificationSettings(types: [.badge, .sound, .alert], categories: nil)
     }
     
-    func didReceiveUserNotificationPermissions(userNotificationPermissionsGranted: UserNotificationPermissionsGranted) {
-        switch userNotificationPermissionsGranted {
-        case .None:
+    func didReceive(userNotificationPermissions: UserNotificationPermissionsGranted) {
+        switch userNotificationPermissions {
+        case .none:
             // The customer has denied user (e.g. local and loud remote) notification permissions.
             break
-        case .Requested:
+        case .requested:
             // The customer has given us permission for all requested user notifications.
             break
-        case let .Some(grantedPermissions):
-            print("Customer refused to give us permissions for \(requestedUserNotificationSettings().types.exclusiveOr(grantedPermissions))")
+        case let .some(grantedPermissions):
```

Comment:
I'm fine with reusing names personally. We have pretty good namespacing for it.

However, "some" here does have different meaning than the "some" in Optional.... basically here it means partial, and in Optional it means it has a value.  I'd only suggest a different name for that reasona alone.